### PR TITLE
Fixed: #6263 V5 (develop) - Missing oauth tables when using the setup wizard

### PR DESCRIPTION
**Pull request by jwhulette**
The issue was related to https://github.com/laravel/passport/issues/483#issuecomment-325790579 .
When setting up passport I run the table migration using the vendor path to the migration files.

### CODE REVIEW

**Comment by snipe** [comment]
Can you do mea favor and skip the php-cs/php-md? It makes it really hard to see what you've changed in these PRs. Thanks for understanding!

File: app/Http/Controllers/SettingsController.php [hunk 1]
```diff
@@ -227,134 +216,136 @@ public function getSetupUser()
...
     {
-
         Artisan::call('migrate', ['--force' => true]);
 
-        $output = Artisan::output();
-
-        if ((!file_exists(storage_path().'/oauth-private.key')) || (!file_exists(storage_path().'/oauth-public.key'))) {
-            Artisan::call('migrate', ['--force' => true]);
+        if ((! file_exists(storage_path() . '/oauth-private.key')) || (! file_exists(storage_path() . '/oauth-public.key'))) {
+            Artisan::call('migrate', ['--path' => 'vendor/laravel/passport/database/migrations']);
```
Comment (jwhulette) [inline app/Http/Controllers/SettingsController.php]: I added this.

**Comment by jwhulette** [comment]
Sorry about that, I removed the PHP_CS file. The only change I made was to the line 249.

File: app/Http/Controllers/SettingsController.php (hunk 1 above)
Comment (snipe) [reply app/Http/Controllers/SettingsController.php]: Ah, got it, thanks!

**Pull request merged**